### ISSUE TITLE
Provide shortcuts for Anki Export dialogue

### ISSUE THREAD

**Issue by artjomsR** [killergerbah/asbplayer#305]
I'm specifically referring to "Open in anki", "Update last card" and "Export". I'm using this dialogue window as my main workflow (as I don't use Yomichan) so it would be nice to have it streamlined as much as possible :)   

![image](https://github.com/killergerbah/asbplayer/assets/6875790/aeb739f4-9328-4eeb-880e-ae7dc3e9d665)

**Comment by killergerbah** [comment]
This should be doable as long as the binds are enabled only when none of the text fields are in focus.

**Comment by artjomsR** [comment]
Ah, that's an interesting limitation. When the dialogue window is first opened, I normally edit the word/definition fields, which I think is a pretty standard use case. Meaning, that by the time I want to invoke `Export` button, one of my fields is already focused.

Is there an easy way to defocus the input fields? Doing so with a mouse defeats the purpose - might as well click the `Export` button directly :) I've tried Esc but it simply closes the popup window :( Maybe it would be possible to override Esc so that it removes the focus from any of the fields, if there are any? And if there are none, then it closes the popup. Not sure if it's too complicated..

**Comment by killergerbah** [comment]
Yes, that could work. Tabbing your way out of the text fields might be another option.

**Comment by artjomsR** [comment]
I think realistically, overriding Esc would be the best option. Tabbing forward from Definition/Word fields would take too many presses. Given the same fields, going backwards is potentially faster, assuming tab focus is removed from the close and settings icons. Not sure if it'd be worth it in the end considering pressing Shift+Tab a few times followed by a shortcut is still probably slower than just using the mouse

**Comment by artjomsR** [comment]
thank you so much!!
